app/src/main/java/com/futurice/freesound/network/api/FreeSoundApi.java

```diff
@@ -38,4 +40,8 @@
     Single<SoundSearchResult> search(@Query("query") @NonNull final String query,
                                      @Query("filter") @Nullable final String filter,
                                      @Query("fields") @NonNull final SoundFields fields);
+
```

Comment:
Yeah, we could probably make some of these constants. Don't worry for now though.